### ISSUE TITLE
Documentation from XML comments doesn't support <see/>

### ISSUE THREAD

**Issue by benmccallum** [ChilliCream/hotchocolate#2159]
**Describe the bug**

Following pure code-first field definition:
```
/// <summary>
/// This field returns a <see cref="MyObject"/>.
/// </summary>
public MyObject MyField { get; set; }
```

Will generate following field description in schema:
`"This field returns a ."`

**To Reproduce**
As above.

**Expected behavior**
Not much we can do to maintain the semantic XML element stuff, but I'd expected it to just render the element's inner text so you'd get `"This field returns a MyObject."`. 

(Note: A slight problem on this is that `MyObject` might not be the graph type name, so a stretch goal would be to notice that and rewrite it to something that makes sense in the schema)

**Desktop (please complete the following information):**
 - OS: Any
 - Version 10.5.0 rc1

**Comment by benmccallum** [comment]
Note, code to look at is probably findable based on this issue and related PR that dealt in this area last:
https://github.com/ChilliCream/hotchocolate/pull/1524
https://github.com/ChilliCream/hotchocolate/issues/1516

**Comment by michaelstaib** [comment]
I gave you write access to the repo... so you should be able now to assign yourself issues, label them and create your own branches without forking.

**Comment by benmccallum** [comment]
Closing as fix in v11 (at least v11.0.8+ though maybe earlier patches too).
